app/(product)/private-cloud/(product)/product/[licencePlate]/layout.tsx

```diff
@@ -0,0 +1,20 @@
+import prisma from '@/lib/prisma';
+
+export default async function Layout({
+  params,
+  edit,
+  decision,
+}: {
+  params: { licencePlate: string };
+  edit: React.ReactNode;
+  decision: React.ReactNode;
+}) {
+  const data = await prisma.privateCloudRequest.findFirst({
+    where: {
+      licencePlate: params.licencePlate,
+      active: true,
+    },
+  });
+
+  return <div className="mt-6">{!data ? edit : decision}</div>;
```

Comment:
I wonder if `layout` page requires `children` to pass into the pages?

Reply:
This is using nextjs parallel routes to conditionally render either the edit or decision page depending on if a active request exists. It's the only way I could find to do this but I think it works nicely. In this case you don't need to render the children but instead the edit and decision pages are passed as props.